packages/ckeditor5-ckbox/src/augmentation.ts

```diff
@@ -24,9 +27,20 @@ declare module '@ckeditor/ckeditor5-core' {
 	interface PluginsMap {
 		[ CKBox.pluginName ]: CKBox;
 		[ CKBoxEditing.pluginName ]: CKBoxEditing;
+		[ CKBoxImageEdit.pluginName ]: CKBoxImageEdit;
+		[ CKBoxImageEditEditing.pluginName ]: CKBoxImageEditEditing;
 	}
 
 	interface CommandsMap {
 		ckbox: CKBoxCommand;
+		ckboxImageEdit: CKBoxImageEditCommand;
 	}
 }
+
+declare global {
```

Comment:
cc: @filipsobol 
Do you know the consequences of moving it here for the projects in TypeScript that uses the editor? Will it interfere with something?

Reply:
I don't expect any issues caused by moving this declaration to this file. I suspect that previous it didn't register at all (or not in all cases), so this is the right move :+1: